desc/perturbations.py

```diff
@@ -760,7 +729,7 @@ def optimal_perturb(  # noqa: C901 - FIXME: break this up into simpler pieces
         if verbose > 0:
             print("Computing d^2g")
         timer.start("d^2g computation")
-        tangents_g = (dxdx_reduced @ dx1_reduced + dxdc @ dc1) @ dxf_dxg
+        tangents_g = dxdx_reduced @ dx1_reduced + dxdc @ dc1
```

Comment:
What happened to `dxf_dxg`?

Reply:
That was basically taking the place of `align_jacobian` which is no longer needed since both objectives take the same args.